.github/workflows/deploy.yml

```diff
@@ -60,6 +60,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           path: ./wheelhouse/*.whl
+          name: aer-deploy-build_wheels_aarch64-${{ matrix.os }}
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Strictly, this set of artifacts (and a couple more noted below) gets uploaded in the line right below this one, so it doesn't need to be pulled down by the `download-artifact` run in the `publish_shared_wheels` job.  Perhaps we could have the prefix be `deploy-shared-`, then jobs that previously uploaded to the `publish-shared-wheels` artifact name can become (e.g.) `deploy-shared-x86_64-${{ matrix.os }}` and `deploy-shared-arm64-macos`, while the others become `deploy-separate-aarch64-linux` etc?  So then the glob is on `deploy-shared-*`, which won't attempt to re-download the wheels that aren't marked as job dependencies.

Reply:
I agree. I changed this to "separate" in d9d7fb78 and set the upload prefix to handle "shared".